WalletWasabi/WabiSabi/Client/DenominationBuilder.cs

```diff
@@ -4,19 +4,20 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using WalletWasabi.Extensions;
 using WalletWasabi.WabiSabi.Backend.Rounds;
 
 namespace WalletWasabi.WabiSabi.Client;
 
 public static class DenominationBuilder
 {
-	public static IOrderedEnumerable<Output> CreateDenominations(Money minAllowedOutputAmount, Money maxAllowedOutputAmount, FeeRate feeRate, bool isTaprootAllowed, Random random)
+	public static IOrderedEnumerable<Output> CreateDenominations(Money minAllowedOutputAmount, Money maxAllowedOutputAmount, FeeRate feeRate, IEnumerable<ScriptType> allowedOutputTypes)
```

Comment:
Probably offtopic, but we plan to add back proper random after this PR. https://github.com/zkSNACKs/WalletWasabi/pull/10859

Do you have anything against that?

Reply:
Nah, that's ok.